cmd/cbft/init_mem.go

```diff
@@ -77,6 +86,32 @@ func initMemOptions(options map[string]string) (err error) {
 	return nil
 }
 
+// Returns total memory based on cgroup limits, if possible.
+func getTotalMemStats() (uint64, error) {
```

Comment:
A better name for this method?

Also, can this be a part of cigar file itself within cbft so that the caller doesn't have to worry about stats.methods?